### PR TITLE
Rails4 rspec

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -6,4 +6,5 @@ require File.expand_path('../config/application', __FILE__)
 
 Hydrus::Application.load_tasks
 
-task :default => [:ci]
\ No newline at end of file
+task(:default).clear
```
Comment (jcoyne) [inline Rakefile]: Please add a comment here about why we need to clear the default.

File: lib/tasks/ci.rake [hunk 2]
```diff
@@ -2,7 +2,11 @@ ZIP_FILE = 'https://github.com/projecthydra/hydra-jetty/archive/v7.3.1.zip'
 
 desc "Run Continuous Integration Suite (tests, coverage, docs)"
 task :ci => ['jetty:clean', 'jetty:config'] do
-  Rails.env = "test"
+  unless Rails.env.test?
+    system('RAILS_ENV=test rake ci')
```
Comment (jcoyne) [inline lib/tasks/ci.rake]: why is this needed? Won't this cause `'jetty:clean', 'jetty:config'` to run twice?

File: spec/support/helpers.rb [hunk 3]
```diff
@@ -0,0 +1,260 @@
+
+# Create a Nokogiri document from an XML source, with some whitespace configuration.
```
Comment (jcoyne) [inline spec/support/helpers.rb]: 👏

File: spec/support/dor_config.rb [hunk 4]
```diff
@@ -0,0 +1 @@
+Dor::Config.configure.suri.mint_ids = false
```
Comment (jcoyne) [inline spec/support/dor_config.rb]: Shouldn't this just be in settings/test.yml?

File: lib/tasks/ci.rake (hunk 2 above)
Comment (cbeer) [inline lib/tasks/ci.rake]: I believe setting `Rails.env` is insufficient for getting our environment correctly set up. We do this same pattern in other places as well (see Searchworks, for one).

Yeah, they'll run twice, but it isn't that expensive.

File: spec/support/dor_config.rb (hunk 4 above)
Comment (cbeer) [inline spec/support/dor_config.rb]: 🤷‍♂️ it was like that when I found it in spec_helper.

Comment (jcoyne) [inline spec/support/dor_config.rb]: Okay, future improvement then?

File: lib/tasks/ci.rake (hunk 2 above)
Comment (cbeer) [inline lib/tasks/ci.rake]: 👌 ❔
